api/src/test/java/ai/djl/nn/norm/GhostBatchNormTest.java

```diff
@@ -0,0 +1,46 @@
+package ai.djl.nn.norm;
+
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.Shape;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class GhostBatchNormTest {
+
+    private GhostBatchNorm gbn;
+    private NDManager manager;
+
+    @BeforeTest
+    void initialize() {
+        this.manager = NDManager.newBaseManager();
```

Comment:
The primary purpose of an NDManager is to manage memory. So, you need to close the manager when you are done using it. You can either do this by adding an `@AfterTest` to contain the close statement or by using a try-with-resources:

```java
try(NDManager manager = NDManager.newBaseManager()) {
  // Do stuff with manager here
}
// manager is automatically closed when exiting the try block
```